ngraph_bridge/ngraph_mark_for_clustering.cc

```diff
@@ -650,7 +657,7 @@ Status MarkForClustering(Graph* graph, const std::set<string> skip_these_nodes,
       set_attributes_map["TopKV2"] = SetStaticInputs({1});
       set_attributes_map["Tile"] = SetStaticInputs({1});
       set_attributes_map["Transpose"] = SetStaticInputs({1});
-
+      set_attributes_map["UnsortedSegmentSum"] = SetStaticInputs({1, 2});
```

Comment:
Both input index 1 and 2 are marked as static inputs. but in the translation only index 2 is being read as static input

Reply:
If you reading index 1 as static only to check for errors then we should not be doing that, coz it will affect the performance

Reply:
marking a node as static is not free. it will cause graph splits (and hence perf issues). We do not want to mark nodes static until absolutely necessary

Reply:
Ok, removed `input 1` from static map.